src/components/portrait/portrait-initials.component.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Browser from '../../utils/helpers/browser';
+
+class PortraitInitials extends React.Component {
+  static propTypes = {
+    /** A custom CSS class for the component. */
+    className: PropTypes.string,
+
+    /** The user's initials to render. */
+    initials: PropTypes.string.isRequired,
+
+    /** The dimensions (size) of the canvas, in pixels. */
+    dimensions: PropTypes.number.isRequired,
+
+    /** Use a dark background. */
+    darkBackground: PropTypes.bool,
+
+    /** The `alt` HTML string. */
+    alt: PropTypes.string
+  }
+
+  static defaultProps = {
```

Comment:
Can we extract from static?
```
PortraitInitials.defaultProps = {}
```

Reply:
See https://github.com/Sage/carbon/pull/2132#discussion_r297175982

Reply:
No longer applicable (`defaultProps` was removed in ee8dd75).